app/assets/javascripts/listings/ListingEligibilityService.js.coffee

```diff
@@ -36,19 +36,19 @@ ListingEligibilityService = ($localStorage, ListingIdentityService, ListingUnitS
     minMax = [1, 1]
     if listing.unitSummary
       min = _.min(_.map(listing.unitSummary, 'minOccupancy')) || 1
-      max = _.max(_.map(listing.unitSummary, 'maxOccupancy')) || Service.defaultMaxOccupancy(amiLevel)
+      max = _.max(_.map(listing.unitSummary, 'maxOccupancy')) || Service.maxNumOfHousehold(amiLevel)
       minMax = [min, max]
     return minMax
 
-  Service.defaultMaxOccupancy = (amiLevel) ->
+  Service.maxNumOfHousehold = (amiLevel) ->
```

Comment:
I think what this function is checking for, is what the max household number that's available on the AMI chart. 

maybe this would be clear if it's like maxAmiNumHousehold or something like that? I can't think of anything better at the moment 🙁